### ISSUE TITLE
ACCOUNT CANCELLATION HELP NEEDED

### ISSUE THREAD

**Issue by mantou10** [koush/vysor.io#1360]
Hi there, 

I was subscribed to Vysor and I was using it for work. Now that I have left the company, I no longer need the account and would like to cancel my account, however, I am unable to do so as my company email has been disabled and I no longer am able to sign in to my Vysor account. 

I am still getting recurring charges of  USD2.50 every month. 

Please help... T_T

**Comment by ahetawal-p** [comment]
Yes same here. I am trying to do the exact same thing.. but there is no way access my account.
